src/formatDate.js

```diff
@@ -50,25 +50,22 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  const outputDateArray = [];
-  const initialDateArray = date.split(fromFormat[3]);
-  const dayIndex = fromFormat.indexOf('DD');
-  const monthIndex = fromFormat.indexOf('MM');
-  const yearIndex = fromFormat.indexOf('YY') === -1
-    ? fromFormat.indexOf('YYYY')
-    : fromFormat.indexOf('YY');
-
-
-
+  const outputDateParts = [];
+  const initialDateParts = date.split(fromFormat[3]);
+  const dayIndex = fromFormat.findIndex(part => part === 'DD');
+  const monthIndex = fromFormat.findIndex(part => part === 'MM');
+  const yearIndex = fromFormat.findIndex(part => part === 'YY') === -1
+    ? fromFormat.findIndex(part => part === 'YYYY')
+    : fromFormat.findIndex(part => part === 'YY');
```

Comment:
```suggestion
  const yearIndex = fromFormat.findIndex(part => part.includes('YY'));
```
Don't repeat yourself

Reply:
Got it) My bad)
Fixed ✔